src/ResourceManagement/Sql/SqlEncryptionProtectorImpl.cs

```diff
@@ -0,0 +1,221 @@
+        }
+
+        ///GENMHASH:6BCE517E09457FF033728269C8936E64:40A980295F5EA8FF8304DA8C06E899BF
+        public SqlEncryptionProtectorImpl Update()
+        {
+            return this;
+        }
+
+        ///GENMHASH:39E79EEEBE800C0263D7785D2CDD0C8F:AA25005E7A9BAF367EA281FE0DB49192
+        public string Uri()
+        {
+            return this.Inner.Uri;
+        }
+
+        ///GENMHASH:5AD91481A0966B059A478CD4E9DD9466:2EA6F22A1E9075415416C51D51B2ED7C
+        protected async Task<Models.EncryptionProtectorInner> GetInnerAsync(CancellationToken cancellationToken = default(CancellationToken))
+        {
+            return await this.sqlServerManager.Inner.EncryptionProtectors
+                .GetAsync(this.resourceGroupName, this.sqlServerName, cancellationToken);
+        }
+
+        ///GENMHASH:7A0398C4BB6EBF42CC817EE638D40E9C:2DC6B3BEB4C8A428A0339820143BFEB3
+        public string ParentId()
+        {
+            var resourceId = ResourceId.FromString(this.Id());
+            return resourceId.Parent.Id;
+        }
+
+        ///GENMHASH:6A5C79A9C5D9A772C2F79EEC7408E4A4:13FE8C3F53ABBA3F908A2B33CEEFD2C4
+        public Models.ServerKeyType ServerKeyType()
+        {
+            return Models.ServerKeyType.Parse(this.Inner.ServerKeyType);
+        }
+
+        ///GENMHASH:2688473C4BAFA54B9FD7ABA11C3C5F8B:D566CA13B0B70C2E92E17CAEA81BBA31
+        public SqlEncryptionProtectorImpl WithServiceManagedServerKey()
+        {
+            this.Inner.ServerKeyName = "ServiceManaged";
+            this.Inner.ServerKeyType = Models.ServerKeyType.ServiceManaged.Value;
+            return this;
+        }
+
+        ///GENMHASH:E24A9768E91CD60E963E43F00AA1FDFE:AC7D8FFF4C733C02A85F39DEEAB80B76
+        public Task DeleteResourceAsync(CancellationToken cancellationToken = default(CancellationToken))
+        {
+            throw new NotImplementedException("Operation not supported");
+        }
+
+        ///GENMHASH:ACA2D5620579D8158A29586CA1FF4BC6:899F2B088BBBD76CCBC31221756265BC
+        public string Id()
+        {
+            return this.Inner.Id;
+        }
+
+        ///GENMHASH:BF5A6EF11B034B3808270F4EF75EB5E0:12F32780E905DD94F4F7DE7FBFB88FCE
+        public SqlEncryptionProtectorImpl WithAzureKeyVaultServerKey(string serverKeyName)
+        {
+            this.Inner.ServerKeyName = serverKeyName;
+            this.Inner.ServerKeyType = Models.ServerKeyType.AzureKeyVault.Value;
+            return this;
+        }
+
+        ///GENMHASH:6A2970A94B2DD4A859B00B9B9D9691AD:6475F0E6B085A35B081FA09FFCBDDBF8
+        public Region Region()
+        {
+            return Microsoft.Azure.Management.ResourceManager.Fluent.Core.Region.Create(this.Inner.Location);
+        }
+
+        ///GENMHASH:507A92D4DCD93CE9595A78198DEBDFCF:4C4C4336C86119672D7AD1E0D4BD29CB
+        public async Task<Microsoft.Azure.Management.Sql.Fluent.ISqlEncryptionProtector> UpdateResourceAsync(CancellationToken cancellationToken = default(CancellationToken))
+        {
```

Comment:
shouldn't this be overriden?

Reply:
Same comment as before, this class inherits from ChildSource<> which does not define this method. This method comes as part of the Java2CSharp conversion.